bin/pfs_integration_test.sh

```diff
@@ -88,33 +88,43 @@ ingestImages.py $TARGET --mode=link \
 # Build bias
 constructBias.py $TARGET --rerun $RERUN/bias \
     --id dateObs=2015-12-22 arm=r spectrograph=1 \
+    --calibId calibDate=2015-12-22 arm=r spectrograph=1 \
     --batch-type=smp --cores $CORES
 genCalibRegistry.py --root $TARGET/CALIB --validity 1000
 ( $CLEANUP && rm -r $TARGET/rerun/$RERUN/bias ) || true
 
 # Build dark
 constructDark.py $TARGET --rerun $RERUN/dark \
     --id dateObs=2015-12-22 arm=r spectrograph=1 \
+    --calibId calibDate=2015-12-22 arm=r spectrograph=1 \
     --batch-type=smp --cores $CORES
 genCalibRegistry.py --root $TARGET/CALIB --validity 1000
 ( $CLEANUP && rm -r $TARGET/rerun/$RERUN/dark ) || true
 
 # Build fiber trace
 constructFiberTrace.py $TARGET --rerun $RERUN/fiber \
     --id visit=104 \
+    --calibId calibDate=2016-11-11 arm=r spectrograph=1 \
     --batch-type=smp --cores $CORES
 genCalibRegistry.py --root $TARGET/CALIB --validity 1000
 ( $CLEANUP && rm -r $TARGET/rerun/$RERUN/fiber ) || true
 
 # Build flat
 constructFiberFlat.py $TARGET --rerun $RERUN/flat \
     --id visit=104..112 \
+    --calibId calibDate=2016-11-11 arm=r spectrograph=1 \
     --batch-type=smp --cores $CORES
 genCalibRegistry.py --root $TARGET/CALIB --validity 1000
 ( $CLEANUP && rm -r $TARGET/rerun/$RERUN/flat ) || true
 
 # Process an arc
-detrend.py $TARGET --rerun $RERUN/detrend --id visit=103
-reduceArc.py $TARGET --rerun $RERUN/detrend --id visit=103
+constructArc.py $TARGET --rerun $RERUN/arc --id visit=103 \
+    --calibId calibDate=2016-11-11 arm=r spectrograph=1 \
+    -c isr.doBias=True isr.doDark=True isr.doFlat=True \
+    isr.doLinearize=False --batch-type none &&
```

Comment:
Are these overrides something that you want every time you run `constructArc.py`? If so, they should go in the obs package or in the definition of `constructArc.py`.